templates/edit_mod.html

```diff
@@ -37,15 +37,16 @@ <h3 style="margin: 0;">{{ error }}</h3>
     {% endif %}
 
     <div class="container lead">
-        <div class="row vertical-centered">
+        <div class="row">
             <div class="col-md-8">
                 <h1 title="{{ mod.name }}">Edit {{ mod.name }}</h1>
                 <input type="text" class="form-control input-block-level" name="short-description" value="{{ mod.short_description }}" placeholder="Short description..." />
             </div>
-            <div class="col-md-2">
+            <div class="col-md-4">
                 <input type="submit" class="btn btn-primary btn-block" value="Save Changes" />
-            </div>
-            <div class="col-md-2">
+                {%- if not mod.published -%}
```

Comment:
I don't remember Jinja templates using `{%-` as delimiter tags, shouldn't it be `%{`?

Also, shouldn't this block be moved a bit up so that it will display `save changes` if the mod is already published, or `save & publish` if the mod isn't published?

Reply:
> I don't remember Jinja templates using `{%-` as delimiter tags, shouldn't it be `%{`?

https://jinja.palletsprojects.com/en/2.11.x/templates/#whitespace-control

> Also, shouldn't this block be moved a bit up so that it will display `save changes` if the mod is already published, or `save & publish` if the mod isn't published?

No, we are not trying to eliminate the existing save button, just add a new button.